api/registrations/serializers.py

```diff
@@ -0,0 +1,52 @@
+from rest_framework import serializers as ser
+from rest_framework import exceptions
+
+from api.base.utils import absolute_reverse
+from api.nodes.serializers import NodeSerializer
+from api.base.serializers import IDField, JSONAPIHyperlinkedIdentityField, LinksField
+
+
+class RegistrationSerializer(NodeSerializer):
+
+    retracted = ser.BooleanField(source='is_retracted', read_only=True,
+        help_text='Whether this registration has been retracted.')
+    date_registered = ser.DateTimeField(source='registered_date', read_only=True, help_text='Date time of registration.')
+
+    registered_by = JSONAPIHyperlinkedIdentityField(
+        view_name='users:user-detail',
+        lookup_field='registered_user_id',
+        link_type='related',
+        lookup_url_kwarg='user_id'
+    )
+
+    registered_from = JSONAPIHyperlinkedIdentityField(
+        view_name='nodes:node-detail',
+        lookup_field='registered_from_id',
+        link_type='related',
+        lookup_url_kwarg='node_id'
+    )
+
+    # TODO: Finish me
+
+    # TODO: Override create?
```

Comment:
I think these notes should stay.

#TODO: finish me -  might refer to OSF-4436 https://openscience.atlassian.net/browse/OSF-4436. Looks like they want to add more fields to the serializer.

#TODO: Override create? might refer to the fact that we can't currently create registrations through the API.  This was something I had worked on in the summer but was put on hold to wait for some pre-reg decisions to be finalized.